src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,82 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size = 0;
```

Comment:
int has default value `0`
```suggestion
    private int size;
```